go.mod

```diff
@@ -80,7 +80,7 @@ require (
 	github.com/rudderlabs/rudder-observability-kit v0.0.3
 	github.com/rudderlabs/rudder-schemas v0.5.0
 	github.com/rudderlabs/sql-tunnels v0.1.7
-	github.com/rudderlabs/sqlconnect-go v1.7.1
+	github.com/rudderlabs/sqlconnect-go v1.7.2-0.20240725064850-4e9872d5ae95
```

Comment:
just curious, how's the package with this version created?

Reply:
Was waiting for the [1.7.2](https://github.com/rudderlabs/sqlconnect-go/pull/145) to be released. Will update the PR once it is merged.

Reply:
https://www.notion.so/rudderstacks/Working-with-multiple-modules-804e0fd7374e4e89a84af64ec9fd170b?pvs=4#5442e7e7ce144a81a55eb92b18392013

Reply:
Based on the commit (`4e9872d5ae956ee7fd8de27e28ecf123e8d836df`) I used, it modified the go.mod accordingly.

Reply:
Upgraded to 1.7.3